contracts/common/GetActionSummary.sol

```diff
@@ -147,17 +144,16 @@ contract GetActionSummary is ExtractCallData, GetActionDomainSkillId {
         totalSummary.domainSkillId = actionSummary.domainSkillId;
       }
 
-      if (isSpecialFunction(actionSummary.sig)) {
-        // If any of the actions are NO_ACTION or OLD_MOVE_FUNDS,
+      if (sigInArray(actionSummary.sig, _dominantSigs)) {
```

Comment:
I would call this function `includes` but that's just a stylistic preference

Reply:
I've renamed it `includes`, and refactored to use [`using...for`](https://docs.soliditylang.org/en/v0.8.26/contracts.html#using-for) syntax so it reads cleanly.